src/main/resources/fabric.mod.json

```diff
@@ -24,7 +24,16 @@
     ]
   },
   "custom": {
-    "fabric-renderer-api-v1:contains_renderer": true
+    "fabric-renderer-api-v1:contains_renderer": true,
+    "modmenu": {
+      "links": {
+        "modmenu.donate": "https://jellysquid.me/donate",
```

Comment:
Should be `https://caffeinemc.net/donate` even if it just redirects currently. We may change it at some point.